minutes/2020-april-30.md

```diff
@@ -0,0 +1,80 @@
+# Meeting Minutes for April 30, 2020
+
+## Agenda
+
+### General
+
+Thomas has created an new private repository (https://github.com/oss-review-toolkit/ort-governance) with the ORT's charter which is basically LF's standard charter with some minor changes. Made the repository private until we sort out the basics of our project.
+
+As the charter is a template there are a couple of open items we as core maintainer have to decide on:
+
+1. Do we want to make any changes to ORT’s charter? 
+
+Thomas read out the charter in the meeting and to TSC colloborated and agreed on below change:
```

Comment:
I believe there's a typo in "colloborated", but I also do not get the meaning in this context. Do you mean "to all TSC members"?